plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -349,23 +349,49 @@ public void showSpecificSecurityWarnings(List<Plugin> plugins) {
         }
     }
 
+    /**
+     * Takes a list of plugins and returns the latest version
+     * Returns existing version if no update
+     * @param plugins updated list of plugins
+     * @return latest plugin versions
+     */
+    public List<Plugin> getLatestVersionsOfPlugins(List<Plugin> plugins) {
```

Comment:
Is this supposed to be part of this PR as well? It's in the other one you submitted as well.

Reply:
it'll be removed from the diff once its merged